### DISCUSSION TITLE
when trying to build using php artisan native:build win i dont know where the application is saving.

### DISCUSSION THREAD

**Discussion by calvin03** [NativePHP/laravel#251] [Q&A]
it says that it should save in dist folder but i cant see any, Im using windows xampp as development

![image](https://github.com/NativePHP/laravel/assets/53732322/7b071dbb-5c5c-41e1-a3c6-cabb92773e1f)

**Comment by simonhamp** [comment]
It's very unlikely that you would have been successful in building for Windows prior to yesterday. If you upgrade to the latest version, you have more luck!

Happy building!
